tipg/settings.py

```diff
@@ -140,13 +141,12 @@ class DatabaseSettings(pydantic.BaseSettings):
     """TiPg Database settings."""
 
     schemas: List[str] = ["public"]
-    exclude_schemas: Optional[List[str]]
     tables: Optional[List[str]]
     exclude_tables: Optional[List[str]]
-    function_schemas: List[str] = ["public"]
-    exclude_function_schemas: Optional[List[str]]
+    exclude_table_schemas: Optional[List[str]]
     functions: Optional[List[str]]
     exclude_functions: Optional[List[str]]
+    exclude_function_schemas: Optional[List[str]]
```

Comment:
**breaking change** 

we now have only one `schemas` option for both `table` and `functions`. To exclude schemas for table or function we can use `exclude_table_schemas` or `exclude_function_schemas` 

cc @bitner

Reply:
we should also add a note somewhere about the importance of the schemas order. like `["myschema", "public"]` do not gives the same result as `["public", "myschema"]`

Reply:
With the commit adding the tipg_get_schemas function, the order of schemas does not matter anymore. The call to pg_table_isvisible meant that for the "landsat" table that was in both public and myschema schemas, it would only ever return one of them into the catalog.